lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -210,8 +210,10 @@ defmodule ExUnit.DocTest do
 
   @doc false
   def __file__(module) do
-    source = module.__info__(:compile)[:source] ||
-             raise "#{inspect module} does not have compile-time source information"
+    source =
+      module.__info__(:compile)[:source] ||
+        raise "#{inspect(module)} does not have compile-time source information"
```

Comment:
wouldn't it be better to use
```
source =
  module.__info__(:compile)[:source]
  || raise "#{inspect(module)} does not have compile-time source information"
```

instead? It would be in line with the way the pipe operator is formatted.

Reply:
There are two ways we format operators, the pipelines, when and `|` are on the next line, all others are on previous.

Reply:
is there a reason behind that? :)

Reply:
The formatter mimics how we are used to write code. While we could probably get used with writing `||` above, I think some other operators, such as `==`, would be weird and some would simply not work `+`.

Reply:
hm makes sense. thanks!